Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,34 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+
+        [TestMethod]
+        public void Configure_CreatesLoggerUsingGivenPath()
+        {
+            // Arrange
+            string targetPath = Path.GetRandomFileName();
+            LogFactory factory = new LogFactory();
+
+            // Act
+            factory.ConfigureLogger(targetPath);
+            FileLogger logger = (FileLogger)factory.CreateLogger("test class");
+
+            // Assert
+            Assert.IsTrue(string.Equals(logger.FilePath, targetPath));
```

Comment:
A better assert would have been to use `Assert.AreEqual`. This way if the assertion fails you get a nice useful error message.
```suggestion
            Assert.AreEqual(targetPath, logger.FilePath);
```